poc_mobile_verifier/src/speedtests.rs

```diff
@@ -408,20 +408,19 @@ impl SpeedtestTier {
 
 #[async_trait::async_trait]
 pub trait SpeedtestStore {
-    async fn fetch(self, id: &PublicKey) -> Result<SpeedtestRollingAverage>;
+    async fn fetch(self, id: &PublicKey) -> Result<SpeedtestRollingAverage, sqlx::Error>;
```

Comment:
This is the part I sorta dislike? It's exposing that it's using sqlx under the hood right?